packages/dapp-wallet/ui/src/components/NavDrawer.jsx

```diff
@@ -1,15 +1,15 @@
 import { useEffect, useState } from 'react';
-import { useMediaQuery, useTheme } from '@material-ui/core';
-import IconButton from '@material-ui/core/IconButton';
-import Menu from '@material-ui/icons/Menu';
-import Drawer from '@material-ui/core/Drawer';
+import { useMediaQuery, useTheme } from '@mui/material';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/icons-material/Menu';
+import Drawer from '@mui/material/Drawer';
 
 import NavMenu from './NavMenu';
 
 const NavDrawer = () => {
   const theme = useTheme();
   const [drawerOpened, setDrawerOpened] = useState(false);
-  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
```

Comment:
Did the breakpoints actually change, or is this unrelated to the upgrade?

Reply:
Yes, they changed the boundary of `breakpoints.down` to be non-inclusive, so you have to increase it by one. They have a script which handled most of these changes https://mui.com/guides/migration-v4/#run-codemods